addons/website/views/assets.xml

```diff
@@ -75,6 +75,7 @@
     </xpath>
     <xpath expr="//script[last()]" position="after">
         <script type="text/javascript" src="/website/static/src/js/set_view_track.js"/>
+        <!-- TODO: Remove the following line after v14.0 - deprecated -->
```

Comment:
As explain by Olivier previously, it can be removed (during the beginning of a freezed version - the only bug will be a potential customization that add a new file after this one especially), it is the file that cannot be delete.